### DISCUSSION TITLE
Use mavlink_msg_home_position to publish home position (instead of mavlink_msg_gps_global_origin)

### DISCUSSION THREAD

**Discussion by Consti10** [iNavFlight/inav#9838] [Ideas]
## Current Behavior
mavlink_msg_gps_global_origin is used to publish the home position. I don't think this is the best fit though (it creates incompability with gcs that work with ardupilot)

## Desired Behavior
INAV shall work just as well with gcs that are also used with ardupilot.

## Suggested Solution
Replace mavlink_msg_gps_global_origin with mavlink_msg_home_position

## Who does this impact? Who is this for?
All inav users using mavlink telemetry.

## Additional context
https://github.com/iNavFlight/inav/pull/9072

**Comment by b14ckyy** [comment]
Moving to Discussion as change request

**Comment by sensei-hacker** [comment]
There is a subtle difference between the two values.

Safehomes and waypoint 255 bring up a question. Origin is of course the takeoff point, and the 0,0 point of the coordinate space. But should home_position be set to the current home (safehome, 255)? Or do we want it to mirror origin?
